tsconfig.json

```diff
@@ -3,22 +3,18 @@
   "compilerOptions": {
     "lib": ["dom", "es2016"],
     "module": "esnext",
-
     /* Output options */
     "noEmit": true,
-
+    "allowUmdGlobalAccess": true,
     /* Strict type-checking options */
     "allowJs": false,
     "strict": true,
     "noFallthroughCasesInSwitch": true,
     "noImplicitReturns": true,
     "noUnusedLocals": true,
-
     /* Module Options */
     "moduleResolution": "node",
-    "esModuleInterop": false,
+    "esModuleInterop": false
   },
-  "include": [
-    "src/**/*.ts",
-  ],
+  "include": ["src/**/*.ts", "node_modules/@webgpu/types"]
```

Comment:
Is this new include necessary? Would expect it to have no effect.

Reply:
not needed.